pom.xml

```diff
@@ -27,151 +32,166 @@
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-xml</artifactId>
 		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-stream-test-support</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-stream-test-support-internal</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-test</artifactId>
+		</dependency>
+
 	</dependencies>
 
-  <properties>
-    <scs-app-maven-plugin.version>1.0.0.BUILD-SNAPSHOT</scs-app-maven-plugin.version>
-    <java.version>1.7</java.version>
-    <spring-cloud-stream.version>1.0.0.RC3</spring-cloud-stream.version>
-    <spring-integration.version>4.2.5.RELEASE</spring-integration.version>
-    <spring-integration-java-dsl.version>1.1.2.RELEASE</spring-integration-java-dsl.version>
-  </properties>
+	<properties>
+		<scs-app-maven-plugin.version>1.0.0.BUILD-SNAPSHOT</scs-app-maven-plugin.version>
+		<java.version>1.7</java.version>
+		<spring-cloud-stream.version>1.0.0.RC3</spring-cloud-stream.version>
+		<spring-integration.version>4.2.5.RELEASE</spring-integration.version>
+		<spring-integration-java-dsl.version>1.1.2.RELEASE</spring-integration-java-dsl.version>
```

Comment:
There is some mess with all these version properties.
See `spring-cloud-stream-app-dependencies`.
I guess we should move all of them into a single place, but I'm not sure what would be better: deps or this root one.